awx/ui/client/lib/components/layout/layout.partial.html

```diff
@@ -76,7 +76,9 @@
         </at-side-nav-item>
     </at-side-nav>
     <div class="at-Layout-main">
-        <ng-transclude></ng-transclude>
+        <div class="at-Layout-content">
+            <ng-transclude></ng-transclude>
+        </div>
         <div class="at-Layout-footer">
```

Comment:
We like the footer when on the login screen, so would you mind adding the following `ng-show`? The change here would be from 
```
<div class="at-Layout-footer">
```
to: 
```
<div class="at-Layout-footer" ng-show="vm.isLoggedIn">
```
Once this is in there, this will be ready for merge. Thanks again!